data-sources/mariadb/config.yml

```diff
@@ -6,14 +6,17 @@ description: |
 icon: logo.svg
 install:
   primary:
+    nerdlet:
```

Comment:
can remove these changes as your marketplace changes are not hosted on master branch.

your marketplace PR: https://source.datanerd.us/nr1-dev-experience/marketplace/pull/968

Reply:
Sure making the changes

Reply:
will get the marketplace pr merged and raise another pr for this changes